### ISSUE TITLE
Remove pining mkdocstrings to 0.17.0

### ISSUE THREAD

**Issue by L1ghtn1ng** [netbox-community/netbox#9163]
### Proposed Changes

Remove the pining of mkdocstrings in base requirements.txt and unpin jinja2 as well

### Justification

There has been a new release of https://github.com/mkdocstrings/mkdocstrings/releases which now supports jinja2 3.1 release

**Comment by DanSheps** [comment]
This issue is pending closure as it does not conform to one of the [provided templates](https://github.com/netbox-community/netbox/issues/new/choose) as required by the [contributing guide](https://github.com/netbox-community/netbox/blob/master/CONTRIBUTING.md). The housekeeping template is **strictly** for maintainers only.  If you'd like to request that your issue be re-opened, please first update the content so that it matches the appropriate template (this may require rewriting your issue entirely).

**Comment by L1ghtn1ng** [comment]
Fair enough just wanted to point it out as that release has been a few months out as it turns out

On 19 Apr 2022, at 15:05, Jeremy Stretch ***@***.***> wrote:

﻿

Thanks for the heads up, but we don't need an issue for this. Dependencies are re-evaluated on each release.
